src/managing-workflow/component-configuration.md

```diff
@@ -19,7 +19,6 @@ env:
     value: "admin_only"
 ```
 
-
```

Comment:
We had a convention of two blank lines before a Markdown header to make it easier to find, just FYI. It's not a big deal either way and we're probably not being consistent, so :shrug:.